ipv6-custom/Makefile

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (C) 2010-2012 OpenWrt.org
```

Comment:
Tom, since this is a new file, would you please add this additional copyright line below the original OpenWrt copyright line:

Modifications from original Copyright (C) 2022 Minim, Inc.

Trying to catch some of this stuff before it goes in.

Thanks,
Rich

Reply:
Done.